coalib/bearlib/abstractions/Linter.py

```diff
@@ -368,6 +379,37 @@ def process_output_corrected(self,
                              diffs={filename: diff},
                              severity=diff_severity)
 
+        def process_output_unidiff(self,
+                                   output,
+                                   filename,
+                                   file,
+                                   diff_severity=RESULT_SEVERITY.NORMAL,
+                                   result_message='Inconsistency found.'):
```

Comment:
good question actually, we could still break the received diff into chunks...

We already had the discussion about meaningful diff-distances for the `corrected` output format, and this is highly non-trivial. Best would be if the program itself emits several patches addressing separate issues, though most tools don't do so (actually I haven't seen a single one).

For simplicity I can accept no diff-distance at all for this PR and emitting the whole patch file as a result. But we need an issue to discuss the introduction of `diff-distance`.